expansion/src/main/java/no/unit/nva/expansion/model/ExpandedResource.java

```diff
@@ -74,12 +73,44 @@ public static ExpandedResource fromPublication(RawContentRetriever uriRetriever,
         return attempt(() -> objectMapper.treeToValue(sortedJson, ExpandedResource.class)).orElseThrow();
     }
 
-    private static JsonNode addFields(String json, Publication publication) {
-        var sortedJson = strToJsonWithSortedContributors(json);
-        injectHasFileEnum(publication, (ObjectNode) sortedJson);
-        expandLicenses(sortedJson);
-        return sortedJson;
-    }
+  private static JsonNode addFields(String json, Publication publication) {
```

Comment:
is it a different codestyle?

Reply:
Yes, unfortunately. The line `injectAnthologyRelation(publication, (ObjectNode) sortedJson);` is new, the rest of `addFields` is unchanged.

Reply:
i see it other places here as well. do you use "UnitStyle" or was it wrong formatting before?

Reply:
I didn't use UnitStyle here, but reformatting with UnitStyle would also cause similar issues...

Reply:
Reformatted the relevant files with UnitStyle now :-)